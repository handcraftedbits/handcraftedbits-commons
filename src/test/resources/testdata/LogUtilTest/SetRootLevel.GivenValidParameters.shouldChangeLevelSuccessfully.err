ERROR | main                 | c.h.c.a.l.LogUtilTest$SetRootLevel$GivenValidParameters | error 
WARN  | main                 | c.h.c.a.l.LogUtilTest$SetRootLevel$GivenValidParameters | warn 
ERROR | main                 | c.h.c.a.l.LogUtilTest$SetRootLevel$GivenValidParameters | error 
WARN  | main                 | c.h.c.a.l.LogUtilTest$SetRootLevel$GivenValidParameters | warn 
