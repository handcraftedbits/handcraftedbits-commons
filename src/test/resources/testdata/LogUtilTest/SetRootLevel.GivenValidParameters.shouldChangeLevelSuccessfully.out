TRACE | main                 | c.h.c.a.l.LogUtilTest$SetRootLevel$GivenValidParameters | trace 
DEBUG | main                 | c.h.c.a.l.LogUtilTest$SetRootLevel$GivenValidParameters | debug 
INFO  | main                 | c.h.c.a.l.LogUtilTest$SetRootLevel$GivenValidParameters | info 
